components/BoundaryLayer.svelte

```diff
@@ -1,30 +1,32 @@
 <script>
   import geojsonExtent from "@mapbox/geojson-extent";
   import mask from "@turf/mask";
-  import { drawPolygon } from "../style.js";
-  import { onMount, getContext } from "svelte";
+  import {
+    drawPolygon,
+    overwriteSource,
+    overwriteLayer,
+  } from "../maplibre_helpers.js";
+  import { getContext } from "svelte";
   import { map } from "../stores.js";
 
   export let boundaryGeojson;
 
   const setCamera = getContext("setCamera");
 
-  onMount(() => {
```

Comment:
Also, the `onMount` stuff isn't helpful in a few of these components. There's nothing we're sticking in the DOM; we're just interacting with the map